cabot/cabotapp/tests/tests_basic.py

```diff
@@ -793,4 +797,47 @@ def test_users_to_notify(self):
     def test_alert(self, fake_send_alert):
         self.service.alert()
         self.assertEqual(fake_send_alert.call_count, 1)
-        fake_send_alert.assert_called_with(self.service, duty_officers=[])
+        fake_send_alert.assert_called_with(self.service, duty_officers=[], fallback_officers=[])
+
+    def test_basic_alert(self):
+        with patch('cabot.cabotapp.models.Service') as mock_service:
+            mock_alert_plugin = Mock(spec=AlertPlugin)
```

Comment:
Would it be simpler to construct a new service object (e.g. at line 87) instead of mocking?